tronbyt_server/utils.py

```diff
@@ -223,3 +224,28 @@ def send_image(
 def push_new_image(device_id: str, logger: logging.Logger) -> None:
     """Wake up WebSocket loops to push a new image to a given device."""
     get_sync_manager(logger).notify(device_id)
+
+
+def server_root() -> str:
```

Comment:
This (and the uses in manager.py) looks like a merge conflict. server_root() and ws_root() were deleted